src/BluetoothManager.cpp

```diff
@@ -179,6 +186,48 @@ void set_imu_characteristics(BNO08x_RVC_Data *heading)
     Serial.write("Read successfully!\n");
 }
 
+void set_imu_characteristics()
+{
+
+    float imu_vals[3] = {DEFAULT_X_ACCEL, DEFAULT_Y_ACCEL, DEFAULT_Z_ACCEL};
+    uint8_t *data = new uint8_t[3 * sizeof(float) + sizeof(uint64_t)];
+    // convert imu_vals to byte array
+    memcpy(data, imu_vals, 3 * sizeof(float));
+    // convert timer to byte array
+    uint64_t temp = timerRead(Timer1);
+    memcpy(data + 3 * sizeof(float), &temp, sizeof(uint64_t));
+    // pYawCharacteristic->setValue(DEFAULT_YAW);
+    // // Serial.write("Yaw updated.\n");
+    // pPitchCharacteristic->setValue(DEFAULT_PITCH);
+    // // Serial.write("Pitch updated.\n");
+    // pRollCharacteristic->setValue(DEFAULT_ROLL);
+    // // Serial.write("Roll updated.\n");
+    // pXAccelCharacteristic->setValue(DEFAULT_X_ACCEL);
+    // // Serial.write("X accel updated.\n");
+    // pYAccelCharacteristic->setValue(DEFAULT_Y_ACCEL);
+    // Serial.write("Y accel updated.\n");
+    pZAccelCharacteristic->setValue(data, 3 * sizeof(float) + sizeof(uint64_t));
+    pZAccelCharacteristic->notify();
+    // Serial.write("Z accel updated.\n");
```

Comment:
We'll need to update these to remove unnecessary characteristics at some point if we are to use a single characteristic for the acceleration data and the timer. We should also add one to hold the yaw, pitch, and roll I think.

Reply:
That's what I was waiting to talk about during the meeting, but I could also do it today while I work on some other things.

Reply:
I think I'm on the same page as you, no need to wait in my mind if you have the time.  Check my understanding. You would be:
- removing all commented out lines
- updating the name of the ZAccelCharacteristic variable
- adding a characteristic to hold the yaw, pitch, and roll